mpi-proxy-split/record-replay.cpp

```diff
@@ -20,11 +20,15 @@
  ****************************************************************************/
 
 #include <mpi.h>
+#ifdef SINGLE_CART_REORDER
 #include <fcntl.h>
+#endif
 #include "jassert.h"
 #include "jconvert.h"
 
+#ifdef SINGLE_CART_REORDER
```

Comment:
Minor: you can make it a single `#ifdef` block by putting `cartesian.h` right under `fcntl.h`.